src/c++/perf_analyzer/report_writer.cc

```diff
@@ -95,7 +96,13 @@ ReportWriter::GenerateReport()
         ofs << "Avg latency,";
       }
       ofs << "request/response,";
-      ofs << "response wait";
+      ofs << "response wait,";
+      if (should_collect_metrics_) {
+        ofs << "Avg GPU utilizations,";
```

Comment:
"Avg GPU Utilization"

Reply:
done

Reply:
this should still be plural (but it should be capitalized). It is a list of per-gpu average utilizations

Reply:
Maybe "Avg GPU Utilization(s)" to be precise?

Reply:
Fine with me for the plurals.  How to do total memories?  Memory(s)? Or just Memory?

Reply:
I think it'd be fine to leave the metric name in the singular. The user could determine through looking at the value that it's a set of values, per GPU, or from reading the documentation.

Reply:
Or name it "
_Abc_Singular_Metric_ Per GPU"

Reply:
Yeah, I agree. We can keep it singular to avoid further complications.